app/audit/AuditDetails.scala

```diff
@@ -16,12 +16,14 @@
 
 package audit
 
-import play.api.libs.json.JsObject
-import play.api.libs.json.Json
-import play.api.libs.json.OWrites
+import play.api.libs.json._
 
-case class AuditDetails(json: JsObject, xml: String)
+case class AuditDetails(json: JsObject)
```

Comment:
Do we even need this class anymore?

Reply:
If you're going to use the same model as us, then I expect this will include the channel field.